pkg/mcs/resourcemanager/server/manager.go

```diff
@@ -141,9 +141,17 @@ func (m *Manager) Init(ctx context.Context) error {
 	}
 
 	// Load keyspace resource groups from the storage.
-	if err := m.loadKeyspaceResourceGroups(); err != nil {
-		return err
-	}
+	// This operation is potentially time-consuming when there are many keyspaces and resource groups.
+	// So we run it in a new goroutine to prevent it from blocking the leader startup.
+	go func() {
+		start := time.Now()
+		log.Info("start to load keyspace resource groups in the background")
+		if err := m.loadKeyspaceResourceGroups(start); err != nil {
```

Comment:
This behavior is different from the old one if the background task failed(maybe due to IO errors). Silently omit the error may cause undefined behavior in some cases.